cypress/test/docs/DocsCards.cy.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { mount } from '@cypress/react';
+import {DocsCards} from '../../../components/docs/DocsCards';
+import IconGettingStarted from '../../../components/icons/GettingStarted';
+import IconTutorials from '../../../components/icons/Tutorials';
+import IconUseCases from '../../../components/icons/UseCases';
+import IconGuide from '../../../components/icons/Guide';
+import IconSpec from '../../../components/icons/Spec';
+
+const cards = [
+    {
+      title: 'Concepts',
+      description: 'Our Concepts section defines the concepts of AsyncAPI features and capabilities.',
+      link: '/docs/concepts',
+      className: 'bg-secondary-200',
+      Icon: IconGettingStarted,
+    },
+    {
+      title: 'Tutorials',
+      description: 'Our Tutorials section teaches beginner processes with AsyncAPI, guiding you from Point A to Point B.',
+      link: '/docs/tutorials',
+      className: 'bg-pink-100',
+      Icon: IconTutorials,
+    },
+    {
+      title: 'Tools',
+      description: 'Our Tools section documents the AsyncAPI tools ecosystem.',
+      link: '/docs/tools',
+      className: 'bg-green-200',
+      Icon: IconUseCases,
+    },
+    {
+      title: 'Guides',
+      description: "Our Guides section teaches AsyncAPI's capabilities at a high level.",
+      link: '/docs/guides',
+      className: 'bg-primary-200',
+      Icon: IconGuide,
+    },
+    {
+      title: 'Reference',
+      description: 'Our Reference section documents the AsyncAPI specification.',
+      link: '/docs/reference',
+      className: 'bg-yellow-200',
+      Icon: IconSpec,
+    }
+  ];
```

Comment:
Please use data buckets here as we use it in DocCards.js. This will maintain consistency among the codebase and we need not to update the things at 2 places.

Reply:
Done , I have resolved the failing test also took the updated data from buckets file 
![image](https://github.com/asyncapi/website/assets/64789514/4152d3e3-f8e1-464a-b2fc-5d7afcc8dff1)